test/anacondalib.py

```diff
@@ -14,11 +14,15 @@
 # along with this program; If not, see <http://www.gnu.org/licenses/>.
 
 import os
+import subprocess
 import sys
-from subprocess import CalledProcessError
+import tempfile
 
 # import Cockpit's machinery for test VMs and its browser test API
 TEST_DIR = os.path.dirname(__file__)
+WEBUI_TEST_DIR = os.path.dirname(__file__)
```

Comment:
That's the same as the existing `TEST_DIR`, why have a different one?

Reply:
Removed.